### ISSUE TITLE
Tested?

### ISSUE THREAD

**Issue by samuk** [tkircher/Espresso_vacuum#1]
This looks nice, has it been fabbed/ tested?

**Comment by tkircher** [comment]
Yes, all interfaces tested successfully, including CSI camera, ethernet, and NVMe SSD on the M.2. Photo of the first test board attached.

The release version has a few minor modifications to clean up the silkscreen and simplify assembly. I moved the HDMI blocking diode from the back to the front, for example, so that a fully functional board with HDMI, USB, MicroSD, and I2C can be reflowed with components on only one side. I'll post photos of that when I get a chance.

![Espresso_Vacuum_v1 0](https://github.com/tkircher/Espresso_vacuum/assets/19326823/e82001a5-6f11-4846-9607-42d433623385)

**Comment by samuk** [comment]
Great! Do you have any plans to sell them anywhere?

**Comment by tkircher** [comment]
No. When it became clear that RPi compute modules were never going to be available again, I designed a new board around an NXP processor instead. I released the project files for this because I have no intention of doing anything more with it.

**Comment by samuk** [comment]
That's a shame. I was interested in them for this board http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-CM4-1.html

I might see how much JLPCB would charge to do a few.
